css-four-column-layout/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="eng">
+
+<head>
+  <title>CSS Four Column</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <div class="container">
+    <div class="row-header">
+      <h2>How It Works</h2>
+    </div>
+    <div class="row-body-half">
+      <div class="col-one-fourth">
+        <img src="/css-four-column-layout/images/screen-1.png" alt="Screen 1 image" class="phone-image">
+        <img src="/css-four-column-layout/images/switch-icon.png" alt="Vector 1 image" class="vector-image">
```

Comment:
Please do not use root paths, use relative paths making use of `..` if you need to move between directories.